schedule/migration/ppc64le_regression_test_online.yaml

```diff
@@ -64,57 +59,15 @@ conditional_schedule:
         - migration/online_migration/online_migration_setup
         - migration/online_migration/register_system
         - migration/online_migration/zypper_patch
-  openldap_to_389ds:
-    REGRESSION_389DS:
-      1:
-        - migration/openldap_to_389ds
   regression_tests:
     REGRESSION_TEST:
       1:
         - locale/keymap_or_locale
-        - console/supportutils
-        - console/force_scheduled_tasks
-        - console/textinfo
         - console/hostname
         - console/upgrade_snapshots
-        - console/x_vt
         - console/zypper_lr
-        - console/pam
-        - console/btrfsmaintenance
-        - console/cpio
-        - console/java
+        - console/check_system_info
         - console/zypper_ref
-        - console/ncurses
-        - console/yast2_lan
-        - console/curl_https
-        - console/salt
-        - console/glibc_tunables
-        - console/zypper_in
-        - console/zypper_log
-        - console/yast2_i
-        - console/yast2_bootloader
-        - console/firewall_enabled
-        - console/krb5
-        - console/sshd
-        - console/ssh_cleanup
-        - console/mtab
         - console/zypper_lifecycle
         - console/orphaned_packages_check
-        - console/kdump_and_crash
-        - console/consoletest_finish
-      2:
         - console/consoletest_finish
-        - x11/desktop_runner
```

Comment:
I think instead of deleting all the x11 test cases, we could at least keep this one, that is simple enough.
`xterm` optionally could also be included, the rest are using those two somehow, wdyt? `setup` perhaps is needed I don't know what that does, perhaps is needed before `xterm`.